src/libs/feature-previews-config.js

```diff
@@ -31,6 +31,13 @@ const featurePreviewsConfig = [
     groups: ['preview-workspace-files'],
     feedbackUrl: `mailto:dsp-sue@broadinstitute.org?subject=${encodeURIComponent('Feedback on workspace files browser')}`,
   },
+  {
+    id: 'hail-batch-azure',
```

Comment:
would like to see this as an exported string so it can be referenced as opposed to present as a raw string in various places that need it